flake8 complains about executorservo.py using bare 'except'

Example: https://travis-ci.org/web-platform-tests/wpt/jobs/466313586#L776

```
py27-flake8 create: /home/travis/build/web-platform-tests/wpt/tools/.tox/py27-flake8
py27-flake8 installdeps: -rrequirements_flake8.txt
py27-flake8 installed: configparser==3.5.0,enum34==1.1.6,flake8==3.6.0,flake8-polyfill==1.0.2,mccabe==0.6.1,pep8-naming==0.7.0,pycodestyle==2.4.0,pyflakes==2.0.0
py27-flake8 run-test-pre: PYTHONHASHSEED='866516015'
py27-flake8 runtests: commands[0] | flake8 --append-config=py27-flake8.ini
./wptrunner/wptrunner/executors/executorservo.py:125:9: E722 do not use bare 'except'
```

The regression was probably introduced by #14242. @jgraham @georgeroman we can fix the code, change revert #14242, or skip this check (in that line or globally). This is currently blocking all changes modifying `tools/` from landing.

--- a/tools/wptrunner/wptrunner/executors/executorservo.py
+++ b/tools/wptrunner/wptrunner/executors/executorservo.py
@@ -122,7 +122,7 @@ class ServoTestharnessExecutor(ProcessTestExecutor):
                     self.proc.wait()
                 else:
                     self.proc.kill()
-        except:
+        except:  # noqa
             self.proc.kill()
             raise
 

